bundler/lib/dependabot/bundler/package_manager.rb

```diff
@@ -12,9 +12,11 @@ module Bundler
     # Keep versions in ascending order
     SUPPORTED_BUNDLER_VERSIONS = T.let([Version.new("2")].freeze, T::Array[Dependabot::Version])
 
-    DEPRECATED_BUNDLER_VERSIONS = T.let([
-      Version.new("1")
-    ].freeze, T::Array[Dependabot::Version])
+    # Currently, we don't support any deprecated versions of Bundler
+    # When we the version is going to be unsupported, we will add it here for a while to give users time to upgrade
+    # Example for deprecation:
+    # DEPRECATED_BUNDLER_VERSIONS = T.let([Version.new("1")].freeze, T::Array[Dependabot::Version])
```

Comment:
👍 very helpful code comment. And we'll definitely want this functionality down the road...

Reply:
Refined comment.